crates/wdk/src/lib.rs

```diff
@@ -42,7 +44,7 @@ pub fn dbg_break() {
     unsafe {
         #[cfg(target_arch = "aarch64")]
         {
-            core::arch::asm!("int 3");
+            core::arch::asm!("brk #0xF000");
```

Comment:
Is this instruction documented somewhere? I couldn't find the asm instruction for breakpoint on Aarch64

Reply:
The documentation is linked in the docstring for this function